TanukiKit/Project.swift

```diff
@@ -98,6 +98,71 @@ public enum VisibilityLevel: Int {
     }
 }
 
+@objc open class Event: NSObject {
+    open var title: String?
+    open var projectID: Int?
+    open var actionName: String?
+    open var targetID: Int?
+    open var targetType: String?
+    open var authorID: Int?
+    open var data: EventData?
+    open var targetTitle: String?
+    open var author: User?
+    open var authorUsername: String?
+    open var createdAt: Date?
+    open var note: EventNote?
+
+    public init(_ json: [String: AnyObject]) {
+        title = json["title"] as? String
+        projectID = json["project_id"] as? Int
+        actionName = json["action_name"] as? String
+        targetID = json["target_id"] as? Int
+        targetType = json["target_title"] as? String
+        authorID = json["author_id"] as? Int
+        data = EventData(json["data"] as? [String: AnyObject] ?? [:])
+        targetTitle = json["target_title"] as? String
+        author = User(json["author"] as? [String: AnyObject] ?? [:])
+        authorUsername = json["author_username"] as? String
+        createdAt = Time.rfc3339Date(string: json["created_at"] as? String)
+        note = EventNote(json["note"] as? [String: AnyObject] ?? [:])
+    }
+}
+
+@objc open class ProjectHook: NSObject {
+    open var id: Int?
+    open var url: URL?
+    open var projectID: Int?
+    open var pushEvents: Bool?
+    open var issuesEvents: Bool?
+    open var mergeRequestsEvents: Bool?
+    open var tagPushEvents: Bool?
+    open var noteEvents: Bool?
+    open var buildEvents: Bool?
+    open var pipelineEvents: Bool?
+    open var wikiPageEvents: Bool?
+    open var enableSSLVerification: Bool?
+    open var createdAt: Date?
+
+    public init(_ json: [String: AnyObject]) {
+        if let id = json["id"] as? Int {
+            self.id = id
+            if let urlString = json["url"] as? String, let parsedURL = URL(string: urlString) {
+                url = parsedURL
+            }
+            projectID = json["project_id"] as? Int
+            pushEvents = json["push_events"] as? Bool
+            issuesEvents = json["issues_events"] as? Bool
+            mergeRequestsEvents = json["merge_requests_events"] as? Bool
+            tagPushEvents = json["tag_push_events"] as? Bool
+            noteEvents = json["ntoe_events"] as? Bool
```

Comment:
Small typo here `ntoe_events` should be `note_events` (`projectHooks` is missing tests)